src/sources/ljm/calibration.py

```
@@ -0,0 +1,190 @@
+        and a list of strings, the names of the addresses set up.
+        """
+
+        list_of_addresses = []
+
+        for sensor in Sensor.sensors:
+            ljm.eWriteName(handle, f"{sensor.channel}_RANGE", sensor.input_range)
+            negChannelValue = ljm.constants.GND
+            if sensor.connection == Connection.DIFFERENTIAL:
+                negChannelValue = CHANNEL_TO_NEGATIVE_CHANNEL[sensor.channel]
+                if negChannelValue == -1:
+                    raise ValueError(f"Invalid negative channel for {sensor.channel}")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)